### ISSUE TITLE
Is there a way I can use this to create a launcher gui with pyqt6

### ISSUE THREAD

**Issue by CherphAlt** [mindstorm38/portablemc#86]
How can I use portablemc to create a gui Minecraft launcher like multimc etc.

**Comment by mindstorm38** [comment]
Yes, of course :) The API is documented [here](https://github.com/mindstorm38/portablemc/blob/master/doc/API.md). The documentation is not complete but covers the basics, you can dive into the source code to discover more. It's also possible to use addons' APIs, such as fabric or forge to install those mod loaders easily.

If you have any question on the API, ask here

**Comment by CherphAlt** [comment]
And also how can I execute custom Minecraft client jar?

**Comment by mindstorm38** [comment]
If you already have a custom version under `.minecraft/versions/`, you can just create a `Version` object with the identifier of your custom version. However, your custom version must have a metadata file (`<version_id>.json`) in the same directory as the JAR file.
